Automattic/woocommerce-services
# Update shrinkwrap respect to latest package.json

Closes https://github.com/Automattic/woocommerce-services/issues/2058

### summary
Update the shrinkwrap file by running `npm run update-deps`

### steps to test
1. pull this branch
2. run `npm ci`
3. create an order
4. try to print a label
5. everything should still work.
